AirCasting/CoreData/PersistenceController+Database.swift

```diff
@@ -108,6 +108,25 @@ extension PersistenceController: SessionRemovable {
     }
 }
 
+extension PersistenceController: SessionUpdateable {
+    func updateSessionUrl(_ url: String, for sessionUUID: SessionUUID, completion: ((Error?) -> Void)?) {
+        let context = self.editContext
+        context.perform {
+            do {
+                let sessionEntity: SessionEntity = try context.newOrExisting(uuid: sessionUUID)
```

Comment:
I'm wondering if we want to use `newOrExisting` here... It's clearly an error when we want to update URL of a non-existent session and by creating new session in that case we're pushing the error further down the line.

Reply:
Good catch, that's definitely true. I'll change that.